src/sagas.js

```diff
@@ -485,8 +483,8 @@ function* getConsoleVm (action) {
  * VmDetail is to be rendered.
  */
 export function* selectVmDetail (action) {
-  yield put(setVmDetailToShow({ vmId: action.payload.vmId }))
   yield fetchSingleVm(getSingleVm({ vmId: action.payload.vmId })) // async data refresh
+  yield put(loadInProgress({ value: false }))
```

Comment:
No, and there is a few reasons:
1. Function fetchAllVms using for a long time, before this commit, without paired with code, because this function is using only in first data loading for showing load component (loadInProgress value has true value by default)
2. Cause of we now has more than one page we shall this use for showing loading component